flake.nix

```diff
@@ -167,7 +178,15 @@
             }
       );
 
-      formatter = forAllSystems (system: (import ./ci { inherit system; }).fmt.pkg);
+      formatter = lib.filterAttrs (
```

Comment:
This hunk is different from the original PR: https://github.com/NixOS/nixpkgs/pull/395520

It removes the formatter from armv6l-linux, riscv64-linux, and x86_64-freebsd because of various eval failures on the individual arches.